multiqc/modules/fastqc/fastqc.py

```diff
@@ -464,6 +470,9 @@ def sequence_content_plot (self):
         <script type="text/javascript">
             if (!window.fastqc_seq_content) fastqc_seq_content = {{}};
             fastqc_seq_content[{module_key}] = {d};
+
+            // Backward compatible global variables
+            window['fastqc_seq_content_data'] = fastqc_seq_content[{module_key}];
```

Comment:
Last commit can be removed if there's no need to keep backward compatibility of global javascript variables